components/infobox/commons/infobox_person_user.lua

```diff
@@ -0,0 +1,99 @@
+---
+-- @Liquipedia
+-- wiki=commons
+-- page=Module:Infobox/Person/User
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local User = require('Module:Infobox/Person')
+local String = require('Module:StringUtils')
+local Class = require('Module:Class')
+
+local Injector = require('Module:Infobox/Widget/Injector')
+local Cell = require('Module:Infobox/Widget/Cell')
+local Title = require('Module:Infobox/Widget/Title')
+local Center = require('Module:Infobox/Widget/Center')
+
+local CustomUser = Class.new()
+
+local CustomInjector = Class.new(Injector)
+
+local _args
+
+function CustomUser.run(frame)
+	local user = User(frame)
+	user.args.informationType = user.args.informationType or 'User'
+	_args = user.args
+
+	user.shouldStoreData = CustomUser.shouldStoreData
+	user.getStatusToStore = CustomUser.getStatusToStore
+	user.getPersonType = CustomUser.getPersonType
+
+	user.createWidgetInjector = CustomUser.createWidgetInjector
+
+	return user:createInfobox(frame)
+end
+
+function CustomInjector:parse(id, widgets)
+	if
+		id == 'history' and
+		not ( String.isEmpty(_args.team_history) and String.isEmpty(_args.clan_history) )
+	then
+		return {
+			Title{ name = 'History' },
+			Center{content = { _args.team_history }},
+			Center{content = { _args.clan_history }},
+		}
+	end
+	return widgets
+end
+
+function CustomInjector:addCustomCells()
+	local widgets = {
+		Cell{name = 'Languages', content = { _args.languages }},
+		Cell{name = 'Favorite players', content = CustomUser:_getArgsfromBaseDefault('fav-player', 'fav-players')},
+		Cell{name = 'Favorite casters', content = CustomUser:_getArgsfromBaseDefault('fav-caster', 'fav-casters')},
+		Cell{name = 'Favorite teams', content = { _args['fav-teams'] }}
+	}
+	if not String.isEmpty(_args['fav-team-1']) then
+		table.insert(widgets, Title{ name = 'Favorite teams' })
+		table.insert(widgets, Center{content = { CustomUser:_getFavouriteTeams() }})
```

Comment:
Notice that the whitespace here is inconsistent. You have brackets both followed by a space and not. I've been using:

```suggestion
		table.insert(widgets, Title{name = 'Favorite teams'})
		table.insert(widgets, Center{content = {CustomUser:_getFavouriteTeams()}})
```